### DISCUSSION TITLE
Boss, please help me

### DISCUSSION THREAD

**Discussion by l289770** [deckerst/aves#1345] [Q&A]
Hello, I deleted some of my photos while using this software, but when I opened it again, I found that the deleted photos had returned. What's going on? Can you solve this problem

**Comment by deckerst** [comment]
From your summary, it's possible that either:
1) they were actually deleted from storage, but are still visible in the app, or
2) they were not actually deleted, so they are still on storage.

You'll need to give way more detailed, logs, etc. if you want to fix your issue.
Please follow the instructions in the app "About" > "Bug Report" if you think there's a bug.

**Reply by l289770** [reply]
Hello, do you know the WeChat avatar? I found that these deleted photos or videos that appeared by themselves or couldn't be deleted at all were in the WeChat avatar. I didn't find the place where this software gave WeChat avatar permission or the photos in WeChat avatar. This software can't operate

**Reply by kmcbest** [reply]
WeChat avatar is part of the program data. When you delete it, the program will download it back from the server in order your avatar (or that of anyone you chat with, or just anyone in any group that you're in where other people chat) can be displayed correctly.
You can check the image location from Aves, and see if I'm right. Then, just hide the album of WeChat program data, and you're good.

微信头像是微信自己的程序数据，你删除了他也会给你下载回来的，不然你聊天的时候就没头像了。所以你可以看看图像详情，是不是微信目录下的，是的话肯定就无法删除了，你可以隐藏微信的目录。
